e2e_tests/tests/api_utils.py

```diff
@@ -27,12 +27,12 @@ def make_session(username: str, password: str) -> api.Session:
 
 @functools.lru_cache(maxsize=1)
 def user_session() -> api.Session:
-    return make_session("determined", conf.USER_PASSWORD)
+    return make_session("determined", "Avalidpassword2")
 
 
 @functools.lru_cache(maxsize=1)
 def admin_session() -> api.Session:
-    return make_session("admin", conf.USER_PASSWORD)
+    return make_session("admin", "Avalidpassword2")
```

Comment:
What are these changes?

Reply:
That was a change from my local e2e test setup, removed this